spec/support/shared_examples/cli/new.rb

```diff
@@ -44,7 +44,6 @@
       # lib/<project>.rb
       #
       expect("lib/#{project}.rb").to have_file_content <<-END
```

Comment:
Good idea with the empty module 👍 
Maybe we also add the comment with this PR? I'm sure that some developers will have questions about this module in future.